searx/webapp.py

```diff
@@ -789,7 +789,8 @@ def search():
             settings['search']['languages'],
             fallback=request.preferences.get_value("language")
         ),
-        timeout_limit = request.form.get('timeout_limit', None)
+        timeout_limit = request.form.get('timeout_limit', None),
+        timings = request.timings
```

Comment:
Is there ever a chance `request.timings` is null, and we may need to do a null check and return an empty list? I don't believe this scenario will happen, but wanted to clarify.

Reply:
The problem is, that the internal `timing` objects are now an official API.  Even this list of `timings` is now a part of the API, including properties such as the order of the list

Since there is unfortunately no documentation for this API, developers now have to look at the code of the templates for every further development ... this massively inhibits any further development.

Unfortunately, we have many such quirks from the history of missing delimitation (API) between active code and templating.

It is also a niche feature that is only used by nerds and DevOps in exceptional cases (analysis); we would now pack analysis data into the HTML response with every response, which unnecessarily sizes the HTML response in 99% of requests.

In my opinion, the runtimes of the engines are analysis data that should be published in a different context, if at all ... I personally think they should not be published at all.